desktop/cmp/form/FormField.js

```diff
@@ -81,7 +83,9 @@ export const [FormField, formField] = hoistCmp.withFactory({
             labelAlign = defaultProp('labelAlign', props, formContext, 'left'),
             labelWidth = defaultProp('labelWidth', props, formContext, null),
             label = defaultProp('label', props, formContext, model?.displayName),
-            commitOnChange = defaultProp('commitOnChange', props, formContext, undefined);
+            commitOnChange = defaultProp('commitOnChange', props, formContext, undefined),
+            tooltipPosition = defaultProp('tooltipPosition', props, formContext, 'right'),
```

Comment:
Think the position should default to 'auto', which is what it was before. That means app devs can take this change without worrying about changes

Reply:
The position used to default to 'right'. See FormField 108-117:

        if (minimal) {
            childEl = tooltip({
                target: childEl,
                targetClassName: `xh-input ${displayNotValid ? 'xh-input-invalid' : ''}`,
                targetTagName: !blockChildren.includes(getReactElementName(child)) || child.props.width ? 'span' : 'div',
                position: 'right',
                disabled: !displayNotValid,
                content: getErrorTooltipContent(errors)
            });
        }

Reply:
Ah! In that case, please ignore my miscomprehension